releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -2,6 +2,23 @@
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
 
+### 5.0.0-preview - December 15, 2021
+* **Breaking Change :** Removed all offline caching capabilities. [#135](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/135)
+* **Breaking Change :** Added new APIs to parse and use sync-token from push notifications received from Event Grid. Using sync-token ensures that users get the latest key-values from App Configuration on any subsequent request. [#278](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/278)
+   ```csharp
+   EventGridEventExtensions.TryCreatePushNotification(this EventGridEvent eventGridEvent, out PushNotification pushNotification)
+   IConfigurationRefresher.ProcessPushNotification(PushNotification pushNotification, TimeSpan? maxDelay = null)
+   ```
```

Comment:
It will be nice to show a code snippet how users should convert from their existing `SetDirty` code to the new one.

Reply:
I think the code snippet will be too long for release notes. I can work with @MBSolomon to update the description of the linked pull request and add the code sample there. Additionally, we'll also update our public documents to use the new APIs.

Reply:
Historically we do have short release notes. But I wouldn't let that limit us. If we have a release worthy of a whole page of notes I don't see a problem with that. Had one like that in .NET feature management: https://github.com/microsoft/FeatureManagement-Dotnet/releases/tag/2.0.0-preview-010610001-1263

Reply:
Okay, I'll add a code snippet here too. Thanks!

Reply:
Added.